core/src/main/java/oracle/weblogic/deploy/logging/WLSDeployLoggingConfig.java

```diff
@@ -156,6 +156,7 @@ public WLSDeployLoggingConfig() {
             System.exit(ERROR_EXIT_CODE);
         }
         PlatformLogger logger = WLSDeployLogFactory.getLogger(WLSDEPLOY_LOGGER_NAME);   // make sure that this is the first logger
+        // this doesn't appear to do anything.  do we need it?
```

Comment:
Please remove this comment and the log.info line below it.

Reply:
Fixed